api/__tests__/app.test.js

```diff
@@ -81,8 +81,84 @@ describe('GET api/articles/', () => {
         })
     })
 
+  test('200: Accepts a topic query ', () => {
+        return supertest(app)
+        .get('/api/articles?topic=mitch')
+        .expect(200)
+        .then(({body}) => {
+
+            expect(body).toHaveLength(4);
 
+            body.forEach((article) => {
+                expect(article.topic).toBe('mitch')
+            })
 
+        })
+        
+    })
+
+    test('400: topic query is invalid', () => {
+        return supertest(app)
+        .get('/api/treasures?topic=banana')
+        .expect(400)
+        .then(({body}) => {
+            expect(body).toEqual({ message: 'Invalid Request'});
+        })
+    })
+
+
+
+
+
+    test('200: Accepts a sort_by ', () => {
+        return supertest(app)
+        .get('/api/articles?sort_by=created_at')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.length).toBe(5);
+            const allSorted = [];
+            body.forEach((item) => {
+                allSorted.push(item.treasure_name)
+            })
+            const sortedArticlesCopy = [...allSorted];
+            const sortedArticles = allSorted.sort()
+            expect(sortedArticles).toEqual(sortedArticlesCopy);
+        })
```

Comment:
I'd recommend using [jest-sorted](https://www.npmjs.com/package/jest-sorted) to make these tests a bit easier